quick_insert.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+# 빠른 테스트 데이터 삽입 스크립트
+
+echo "============================================"
+echo "Studiz 서버 테스트 데이터 삽입"
+echo "============================================"
+
+# EC2 서버 정보
+EC2_HOST="3.27.86.20"
+EC2_USER="ec2-user"
+SSH_KEY="~/.ssh/studiz-server-key.pem"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)